iowrite.cpp

```diff
@@ -1373,6 +1373,18 @@ bool writeRestart(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpiGrid,
    Writer vlsvWriter;
    const int masterProcessId = 0;
    MPI_Info MPIinfo;
+   if (P::restartWriteHints.size() == 0) {
+      MPIinfo = MPI_INFO_NULL;
+   } else {
+      MPI_Info_create(&MPIinfo);
+
+      for (std::vector<std::pair<std::string,std::string>>::const_iterator it = P::restartWriteHints.begin();
+           it != P::restartWriteHints.end();
+           it++)
+      {
+         MPI_Info_set(MPIinfo, it->first.c_str(), it->second.c_str());
+      }
+   }
    if (stripe == 0 || stripe < -1){
```

Comment:
now if stripe is zero or smaller than -1, the manually provided hints will be removed.

Reply:
also on line 1192 for bulk files.

Reply:
So the subsequent `MPI_Info_set` call would override the previous `MPI_Info_set(MPIinfo, it->first.c_str(), it->second.c_str());` calls?

Reply:
No OK sorry I read too quickly.